### PR TITLE
update contributor information - daniel saunders

### PR DESCRIPTION
**Pull request by daniel-saunders-phil**
I added a picture to the folder and updated my information to include a link to that picture. Mostly I'm just trying to see how pull requests works :)

### CODE REVIEW

File: 2022-07_sprint/sprint_parties/contributors.md [hunk 1]
```diff
@@ -447,7 +447,7 @@ Daniel Saunders
 :::
 :::{grid-item}
 
-<img alt="Image for contributor template" src="../../_static/contributors/blank_person.jpg" class="rounded-circle">
+<img alt="Image for contributor template" src="../../_static/contributors_2022_07/daniel_saunders.jpg" class="rounded-circle">
```
Comment (OriolAbril) [inline 2022-07_sprint/sprint_parties/contributors.md:450]: As you can see in the preview: https://pymc-data-umbrella--145.org.readthedocs.build/en/145/2022-07_sprint/sprint_parties/contributors.html if you scroll down, you are modifying the image for Yuan Li, not your own.

All the info for each contributor: image, name, location, social links; is inside the same `grid-item`. That is, below a `:::{grid-item}` opening and a `:::` closing, each `grid-item` enclosing corresponds to a different contributor

**Review by OriolAbril — CHANGES_REQUESTED** [review]

**Comment by reshamas** [comment]
Hi @daniel-saunders-phil ! Good to see you here!

If you see above after "All checks have passed", the "docs/..." has a link to a preview:
https://pymc-data-umbrella--145.org.readthedocs.build/en/145/2022-07_sprint/sprint_parties/contributors.html

You can see in this preview that you changed Yuan Li's photo instead of your own.
Want to give it another try?

#DataUmbrellaPyMCSprint

File: 2022-07_sprint/sprint_parties/contributors.md (hunk 1 above)
Comment (daniel-saunders-phil) [reply 2022-07_sprint/sprint_parties/contributors.md]: oops! I see my mistake now. Thanks for explaining that.

**Comment by daniel-saunders-phil** [comment]
Hi thanks for the welcome! I made the adjustment and I checked the preview with the link this time. I imagine I just resubmit the request at this point?

**Review by OriolAbril — APPROVED** [review]

**Comment by OriolAbril** [comment]
Thanks!

**Pull request merged**